ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host2.md

```diff
@@ -1418,9 +1418,9 @@ router pim sparse-mode
 
 #### 802.1X Global
 
-| System Auth Control | Protocol LLDP Bypass | Dynamic Authorization |
-| ------------------- | -------------------- | ----------------------|
-| True | True | True |
+| System Auth Control | Protocol LLDP Bypass | Dynamic Authorization | Dropped Statistics Packets |
```

Comment:
```suggestion
| System Auth Control | Protocol LLDP Bypass | Dynamic Authorization | Dropped Packets Statistics |
```

Reply:
Those two are not the same. Are we counting how many statistics we dropped or are we doing statistics over dropped packets?